Console.Calculator.oksanatk/CalculatorLibrary.cs

```diff
@@ -0,0 +1,189 @@
+﻿using Newtonsoft.Json;
+using Spectre.Console;
+
+namespace CalculatorLibrary
```

Comment:
🟠 **Namespaces**
As per the academy's [code-conventions](https://thecsharpacademy.com/article/58/code-conventions) - Use the newest namespaces.
Use file-scoped namespaces to save an indentation level:

❌ 
```
// Block-Scoped Namespace
namespace YourNamespace
{
    // ...
}
```

✔️ 
```
// File-Scoped Namespace
namespace YourNamespace;

// ...
```

Reply:
Hey Chris,

I really appreciate you specifying the namespace when explaining what was lacking in my coding conventions. 

I've received the feedback that my coding conventions were missing something before, but I guess I was so busy trying to see if I had any places with not enough blank lines, or too many, that I missed something so obvious.

I'm now at the point of reviewing the next project for coding conventions, and that was definitely something that I've paid attention to.

You probably don't remember by now, but I'll ask just in case if you happened to have made notes when you looked over this project. Were there any other Code-Convention / Style Guide Compliance violations that jumped out at you?

Many thanks,
Oksana

Reply:
Hey @oksanatk 👋 ,

No worries! I have added some comments,

overall your code style is good! I have been very nit-picky in my comments to try help!

Hope it helps!

@chrisjamiecarter